src/showcase/components/button/TextRevelButton.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import React from 'react'
+
+const TextRevelButton = () => {
```

Comment:
Spelling Mistake here, should be `TextRevealButton`

path will also be `button/TextRevealButton`

Reply:
fixed